README.md

```diff
@@ -10,14 +10,8 @@
 $ cargo install cargo-fuzz
 ```
 
-Note: `libFuzzer` needs LLVM sanitizer support, so this only works on x86-64 Linux, x86-64 macOS
-and Apple-Silicon (aarch64) macOS for now. You'll also need a C++ compiler with C++11 support.
-
-If you have an old version of `cargo fuzz`, you can upgrade with this command:
-
-```sh
-$ cargo install -f cargo-fuzz
-```
+Note: `libFuzzer` needs LLVM sanitizer support, so this only works on x86-64 and Aarch64,
+and only on Unix-lile operating systems (not Windows). You'll also need a C++ compiler with C++11 support.
```

Comment:
```suggestion
and only on Unix-like operating systems (not Windows). You'll also need a C++ compiler with C++11 support.
```

Reply:
Sure thing. Thanks for the quick review!